doc/classes/Vector2.xml

```diff
@@ -329,7 +329,7 @@
 		<method name="sign" qualifiers="const">
 			<return type="Vector2" />
 			<description>
-				Returns a new vector with each component set to one or negative one, depending on the signs of the components, or zero if the component is zero, by calling [method @GlobalScope.sign] on each component.
+				Returns this vector with each component set to 1.0 or -1.0, depending on the sign of that component, or 0.0 if the component is zero. The result is identical to calling [method @GlobalScope.sign] on each component.
```

Comment:
This is a const methods, it cannot by definition return _this_ vector, it creates a new one (and the implementation of it confirms this). I suspect the same is true for other methods that you've edited in a similar manner? At the very least, worth a second check.

```cpp
Vector2 Vector2::sign() const {
	return Vector2(SIGN(x), SIGN(y));
}
```

Please also wrap code literals in `[code][/code]`, e.g. `[code]1.0[/code]`.

Reply:
If there are other related methods that have this issue in their descriptions, I think it's okay to update the docs in this PR. You're adding more such methods, so it's totally appropriate! But yeah, we need to be careful with the wording.